skit.API/Controllers/Areas/Companies/CompaniesController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using skit.Application.Companies.Commands.DeleteCompany;
+using skit.Application.Companies.Commands.UpdateCompany;
+using skit.Application.Companies.Queries.BrowseCompanies;
+
+namespace skit.API.Controllers.Areas.Companies;
+
+[Route($"{Endpoints.BaseUrl}/companies")]
+public class CompaniesController : BaseController
+{
+    [HttpGet]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public async Task<ActionResult<List<BrowseCompaniesQuery>>> BrowseCompanies([FromQuery] BrowseCompaniesQuery query, CancellationToken cancellationToken = default)
+    {
+        var response = await Mediator.Send(query, cancellationToken);
+        return Ok(response);
+    }
+    
+    [HttpPut("{companyId:guid}")]
```

Comment:
companyId powinno być brane z claima zalogowanego usera, ale że tego jeszcze nie ma to na razie jest ok, ja to potem poprawię jak zrobić logowanie i rejestrację